authentication/auth.go

```diff
@@ -779,6 +781,7 @@ func GetDefaultValues() map[string]string {
 	defaults["PARTICIPANT_ID"] = PARTICIPANT_ID
 	defaults["FIRST_NAME"] = "John"
 	defaults["TEST_QUESTIONS"] = "F"
+	defaults["sds_dataset_id"] = "001"
```

Comment:
dataset id will be a uuid4

Reply:
The type is right, but they haven't defined a format. UUID4 is a string, it is a string in a specific format.

Reply:
now uses a uuid so resolving 👍